draft-irtf-cfrg-aead-limits.md

```diff
@@ -678,45 +706,209 @@ IA <= AEA
 {{mu-gcm-ae}} therefore contains the integrity limits.
 
 
-## AEAD_CHACHA20_POLY1305, AEAD_AES_128_CCM, and AEAD_AES_128_CCM_8
+## AEAD_CHACHA20_POLY1305
 
-There are currently no concrete multi-key bounds for AEAD_CHACHA20_POLY1305,
-AEAD_AES_128_CCM, or AEAD_AES_128_CCM_8. Thus, to account for the additional
-factor `u`, i.e., the number of keys, each `p` term in the confidentiality and
-integrity limits is replaced with `p / u`.
+Concrete multi-key bounds for AEAD_CHACHA20_POLY1305 are given in Theorem 7.8
+in {{ChaCha20Poly1305-MU}}, covering protocols with nonce randomization like
+TLS 1.3 {{TLS}} and QUIC {{?RFC9001}}.
+
+For this AEAD, n = 512, k = 256, t = 128, and r = 96; the length l is the sum
+of AAD and plaintext (in blocks of 128 bits).
+
+### Authenticated Encryption Security Limit {#mu-ccp-ae}
+
+<!--
+    From {{ChaCha20Poly1305-MU}} Theorem 7.8; for nonce randomization (XN transform).
+    
+    Let:
+        - d: the max. number of times any nonce is repeated across users
```

Comment:
The paper's complexity indeed isn't helping with extracting the simplified bounds we want to present here *sigh

\delta is a formal tuning parameter in the underlying stochastic "balls-into-bins" lemma, which plays a role when determining d: the number of times a nonce is repeated across users. This balls-into-bins can be used to determine d because of how nonces are formed with randomization.
Now: the larger \delta, the smaller the 8th term [correct], but also the larger d.

In the paper's discussion, we chose \delta = 2 simply as this makes the 8th term "small enough" to not dominate. 1/2 would be mimicking the GGM analysis, and lead to the 8th term being 2^-48 ("problematically large"). Other values might be reasonable, too; based on what Chris said, we should discuss this.